compiler/rustc_codegen_llvm/src/abi.rs

```diff
@@ -74,19 +80,14 @@ impl ArgAttributesExt for ArgAttributes {
             if let Some(align) = self.pointee_align {
                 llvm::LLVMRustAddAlignmentAttr(llfn, idx.as_uint(), align.bytes() as u32);
```

Comment:
The alignment can be ABI-relevant if the argument is byval. For simplicity, probably best to always add it.